languages/ruby/lib/oso/polar/ffi/message.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Oso
+  module Polar
+    module FFI
+      # Wrapper class for Message FFI pointer + operations.
+      class Message < ::FFI::AutoPointer
+        # @return [String]
+        def to_s
+          @to_s ||= read_string.force_encoding('UTF-8')
+        end
+
+        Rust = Module.new do
+          extend ::FFI::Library
+          ffi_lib FFI::LIB_PATH
+
+          attach_function :free, :string_free, [Message], :int32
+        end
+
+        def process
+          message = JSON.parse(to_s)
+          kind = message['kind']
+          msg = message['msg']
+
+          case kind
+          when 'Print'
+            puts(msg)
+          when 'Warning'
+            puts('[warning] %<msg>s')
```

Comment:
Shouldn't this be
```suggestion
            $stderr.puts('[warning] %<msg>s')
```
to match the other language library implementations?

Reply:
rubocop says `warn`

Reply:
TIL